doc/governance.rst

```diff
@@ -21,6 +21,8 @@ Bodies
 Ceph Executive Council
 ======================
 
+See :ref:`Ceph Executive Council Responsibilities <ceph-executive-council-responsibilities>`.
```

Comment:
"Responsibilities in practice"

Reply:
I've made this change. I've also repaired all the scraggly loose ends that were preventing the RST from cleanly building.